packages/instantsearch.js/src/connectors/voice-search/connectVoiceSearch.ts

```diff
@@ -106,14 +106,15 @@ const connectVoiceSearch: VoiceSearchConnector = function connectVoiceSearch(
               const queryLanguages = language
                 ? [language.split('-')[0]]
                 : undefined;
+              // @ts-ignore queryLanguages is allowed to be a string, not just an array
```

Comment:
same here if you confirm I can fix the spec

Reply:
same I believe it's allowed in the api just because any array is also allowed to be a string, I'd rather leave it to be an array always and eventually change it in InstantSearch